pages/faq/index.tsx

```diff
@@ -0,0 +1,132 @@
+import Head from 'next/head'
+import {
+  Typography,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Grid,
+} from '@material-ui/core/'
+import { CopyHolder } from '../../types/language'
+import useLanguage from '../../hooks/useLanguage'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import {
+  faqCopy,
+  whereToStart,
+  probation,
+  transferProbation,
+  treatment,
+  refill,
+  covid,
+  documents,
+  useStyles,
+  siteTitle,
+  volunteer,
+} from '../../constants/'
+const FaqPage = () => {
+  const { language } = useLanguage()
+  const classes = useStyles()
+  const faqs: CopyHolder[] = [
+    whereToStart,
+    probation,
+    transferProbation,
+    volunteer,
+    treatment,
+    refill,
+    covid,
+    documents,
+  ]
+  const activeCopy = faqCopy[language]
+  return (
+    <div className={classes.root}>
+      <Head>
+        <title>{`${siteTitle} | ${activeCopy.title}`}</title>
+      </Head>
+      <Typography
+        style={{ marginTop: '3rem' }}
+        align="center"
+        variant="h2"
+        component="h2"
+      >
+        {activeCopy.title}
+      </Typography>
+      <Typography
+        style={{
+          marginTop: '2rem',
+          marginBottom: '2rem',
+          padding: '2rem',
+          textAlign: 'justify',
+        }}
+        align="center"
+        variant="h4"
+        component="h4"
+      >
+        {activeCopy.description}
+      </Typography>
+      <div>
+        {' '}
+        {faqs.map((item, key) => {
+          const activeCopy = item[language]
+          return (
+            <Accordion key={key} style={{ margin: '1rem' }}>
+              <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="panel3a-content"
+                data-testid="accordion"
+                id="panel3a-header"
+              >
+                <Typography>
+                  {' '}
+                  <p className={classes.heading}>{activeCopy.listItem}</p>
+                </Typography>
+              </AccordionSummary>
+              <AccordionDetails>
+                <Grid container spacing={3}>
+                  <Grid item xs={12}>
+                    <Typography>
+                      {activeCopy && activeCopy.href1 && (
+                        <a
```

Comment:
@vsauceda-cfa I think that we should use `NextLink` or `push` for navigation within the app. Using an `a` tag will create confusion and introduce complexity down the line if, for example, someone wanted to include analytics in Thrive.

Also, for links that have `click here` or something similar in them, what do you think of having a traditional underlined text link instead of having the entire item be the link? I think that may introduce less confusion and better follow the design patterns we've established so far.